docs/ota-client-guide/modules/ROOT/pages/posix-secondaries-bitbaking.adoc

```diff
@@ -99,7 +98,7 @@ For example,
 
 It is assumed that a reader is a familiar with bitbaking for Raspberry Pi in general, see xref:build-raspberry.adoc[Build a Raspberry Pi image].
 
-The aforementioned/above guide is relevant and applicable to building a Raspberry images of Primay and Secondary.
+The aforementioned/above guide is relevant and applicable to building a Raspberry Pi images of Primary and Secondary.
```

Comment:
`to building a Raspberry Pi images` - I suppose this `a` article should not be there?

Reply:
Fixed, thanks.